sections/Deals/CreateDeal.tsx

```diff
@@ -264,6 +264,14 @@ const CreateDeal: FC<CreateDealProps> = ({ poolAddress, purchaseToken }) => {
 		};
 	}, [formik?.values, poolAddress, provider, signer, walletAddress]);
 
+	const underlyingDealTokenTotal = useMemo(() => {
+		if (!formik.values?.purchaseTokenTotal || formik.values.dealExchangeRate <= 0) {
```

Comment:
why would the dealExchangeRate be less than 0? what if it is an empty string?

Reply:
It's the input value so the user can enter any number